site/snippets/header.php

```diff
@@ -18,5 +18,61 @@
   <?php echo js('js/libs/modernizr.js') ?>
   <?php echo js('js/main.js') ?>
 
+
+<!-- jQuery for the Contact Form-->
+  <script type="text/javascript">
+$(document).ready(function() {
+    $("#submit_btn").click(function() { 
+       
+      var proceed = true;
+        //simple validation at client's end
+        //loop through each field and we simply change border color to red for invalid fields   
+    $("#contact_form input[required=true], #contact_form textarea[required=true]").each(function(){
+      $(this).css('border-color',''); 
+      if(!$.trim($(this).val())){ //if this field is empty 
+        $(this).css('border-color','red'); //change border color to red   
+        proceed = false; //set do not proceed flag
+      }
+      //check invalid email
+      var email_reg = /^([\w-\.]+@([\w-]+\.)+[\w-]{2,4})?$/; 
+      if($(this).attr("type")=="email" && !email_reg.test($.trim($(this).val()))){
+        $(this).css('border-color','red'); //change border color to red   
+        proceed = false; //set do not proceed flag        
+      } 
+    });
+       
+        if(proceed) //everything looks good! proceed...
+        {
+      //get input field values data to be sent to server
+            post_data = {
+        'user_name'   : $('input[name=name]').val(), 
+        'user_email'  : $('input[name=email]').val(), 
+        'user_org'    : $('input[name=org]').val(), 
+        'msg'     : $('textarea[name=message]').val()
+      };
+            
+            //Ajax post data to server
+            $.post('contact_me.php', post_data, function(response){  
```

Comment:
This is why its not working. Its the wrong path.  It should '../templates/contact_me.php'